ui/src/components/ResourceDialog/ProcessorsDialog/ProcessorForm.tsx

```diff
@@ -48,7 +48,7 @@ export const ProcessorForm: React.FC<Props> = ({
         </Grid>
       </Grid>
       <form style={{ flexGrow: 1, overflow: "auto" }}>
-        <Grid container spacing={3} className={mixins["mb-5"]}>
+        <Grid container spacing={3} className={mixins["mb-5"]} marginTop="-10px">
```

Comment:
Could we do this with paddingTop instead of a negative margin?

Negative margins aren't really bad if they're required but I think this is more correct in this case

```suggestion
        <Grid
          container
          spacing={3}
          className={mixins["mb-5"]}
          paddingTop={"8px"}
        >
```